releasenotes/notes/fix-quasi-probabilty-zero-padding-1a09a41eecb19b12.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
```

Comment:
You don't need a release note this hasn't been released yet so it's covered by the new feature release note